src/Client/BuzzHouse/Generator/ExternalIntegrations.h

```diff
@@ -249,7 +281,24 @@ class MySQLIntegration : public ClickHouseIntegratedDatabase
         }
     }
 
-    std::string getTableName(const uint32_t tname) override { return "test.t" + std::to_string(tname); }
+    std::string getTableName(std::shared_ptr<SQLDatabase> db, const uint32_t tname) override
+    {
+        std::string res;
+
+        if (is_clickhouse && db)
+        {
+            res += "d";
+            res += std::to_string(db->dname);
+            res += ".";
+        }
+        else if (!is_clickhouse)
+        {
+            res += "test.";
+        }
+        res += "t";
+        res += std::to_string(tname);
+        return res;
```

Comment:
This is very verbose for what it needs to do, IMHO. Other parts of the code are similar, with `if` branches and lots of `+=` for std::string. First, we already have a `String` defined which is just a `std::string`. We shouldn't be using `std::string` but `String`. Please stick to `String` as much as possible.

Also, why don't we use simpler stuff through `fmt::format` to simplify the logic?
```cpp
        auto prefix = is_clickhouse ? (db ? fmt::format("d{}.", db->dname) : "") : "test.";
        return fmt::format("{}t{}", prefix, tname);
```

Reply:
I did this way to avoid allocations. In your approach, `fmt::format` always returns a new string.

Reply:
Well, to be fair you could call `fmt::format` only once at the expense of making the code less readable.
In your version you're actually using at least 2 allocations and probably 3 (each `std::to_string` + in case it gets into any if branch). I said at least because the original `res` String will have SSO, but depending on the size of the names there could be an extra realloc.